gravitee-apim-repository/gravitee-apim-repository-redis/pom.xml

```diff
@@ -64,36 +64,19 @@
 		</dependency>
 
 		<dependency>
-			<groupId>jakarta.activation</groupId>
-			<artifactId>jakarta.activation-api</artifactId>
-			<scope>provided</scope>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-redis-client</artifactId>
 		</dependency>
 
 		<dependency>
-			<groupId>jakarta.annotation</groupId>
-			<artifactId>jakarta.annotation-api</artifactId>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-core</artifactId>
 			<scope>provided</scope>
 		</dependency>
 
 		<dependency>
-			<groupId>io.lettuce</groupId>
-			<artifactId>lettuce-core</artifactId>
-			<version>${lettuce.version}</version>
-		</dependency>
-
-		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-lang3</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-pool2</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-api</artifactId>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-rx-java2</artifactId>
```

Comment:
we are doing some change to use vertx-rx-java3 💥

Reply:
I update this version.